Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -868,98 +892,102 @@ variable {R' A : Type _} [Semiring R'] [Semiring A] [∀ i, Module A (M₁ i)] [
 /-- The space of multilinear maps over an algebra over `R` is a module over `R`, for the pointwise
 addition and scalar multiplication. -/
 instance [Module R' M₂] [SMulCommClass A R' M₂] : Module R' (MultilinearMap A M₁ M₂) where
-  add_smul r₁ r₂ f := ext fun x => add_smul _ _ _
-  zero_smul f := ext fun x => zero_smul _ _
+  add_smul _r₁ _r₂ _f := ext fun _x => add_smul _ _ _
+  zero_smul _f := ext fun _x => zero_smul _ _
 
 instance [NoZeroSMulDivisors R' M₃] : NoZeroSMulDivisors R' (MultilinearMap A M₁ M₃) :=
-  coe_injective.NoZeroSMulDivisors _ rfl coe_smul
+  coe_injective.noZeroSMulDivisors _ rfl coe_smul
 
 variable (M₂ M₃ R' A)
 
-/-- `multilinear_map.dom_dom_congr` as a `linear_equiv`. -/
+/-- `MultilinearMap.domDomCongr` as a `LinearEquiv`. -/
 @[simps apply symm_apply]
 def domDomCongrLinearEquiv {ι₁ ι₂} (σ : ι₁ ≃ ι₂) :
-    MultilinearMap A (fun i : ι₁ => M₂) M₃ ≃ₗ[R'] MultilinearMap A (fun i : ι₂ => M₂) M₃ :=
-  {
-    (domDomCongrEquiv σ :
-      MultilinearMap A (fun i : ι₁ => M₂) M₃ ≃+ MultilinearMap A (fun i : ι₂ => M₂) M₃) with
+    MultilinearMap A (fun _i : ι₁ => M₂) M₃ ≃ₗ[R'] MultilinearMap A (fun _i : ι₂ => M₂) M₃ :=
+  { (domDomCongrEquiv σ :
+      MultilinearMap A (fun _i : ι₁ => M₂) M₃ ≃+ MultilinearMap A (fun _i : ι₂ => M₂) M₃) with
     map_smul' := fun c f => by
       ext
-      simp }
+      simp [MultilinearMap.domDomCongr] }
 #align multilinear_map.dom_dom_congr_linear_equiv MultilinearMap.domDomCongrLinearEquiv
-
+#align multilinear_map.dom_dom_congr_linear_equiv_apply MultilinearMap.domDomCongrLinearEquiv_apply
+#align multilinear_map.dom_dom_congr_linear_equiv_symm_apply MultilinearMap.domDomCongrLinearEquiv_symm_apply
 variable (R M₁)
 
-/-- The dependent version of `multilinear_map.dom_dom_congr_linear_equiv`. -/
+/-- The dependent version of `MultilinearMap.domDomCongrLinearEquiv`. -/
 @[simps apply symm_apply]
 def domDomCongrLinearEquiv' {ι' : Type _} (σ : ι ≃ ι') :
     MultilinearMap R M₁ M₂ ≃ₗ[R] MultilinearMap R (fun i => M₁ (σ.symm i)) M₂ where
   toFun f :=
     { toFun := f ∘ (σ.piCongrLeft' M₁).symm
-      map_add' := fun _ m i => by
+      map_add' := fun m i => by
         skip
-        letI := σ.decidable_eq
+        letI := σ.decidableEq
         rw [← σ.apply_symm_apply i]
         intro x y
-        simp only [comp_app, Pi_congr_left'_symm_update, f.map_add]
-      map_smul' := fun _ m i c => by
+        simp only [comp_apply, piCongrLeft'_symm_update, f.map_add]
+      map_smul' := fun m i c => by
         skip
-        letI := σ.decidable_eq
+        letI := σ.decidableEq
         rw [← σ.apply_symm_apply i]
         intro x
-        simp only [comp_app, Pi_congr_left'_symm_update, f.map_smul] }
+        simp only [Function.comp, piCongrLeft'_symm_update, f.map_smul] }
```

Comment:
Inconsistent here too, probably my fault. It looks like we're missing `#align comp_app Function.comp_apply`